projects/npm-tools/packages/npm-scripts/src/jest/mocks/Liferay.js

```diff
@@ -158,14 +223,22 @@ const Util = {
 	 * https://github.com/liferay/liferay-portal/blob/31073fb75fb0d3b309f9e0f921cb7a469aa2703d/modules/apps/frontend-js/frontend-js-aui-web/src/main/resources/META-INF/resources/liferay/util.js#L999
 	 */
 	sub: jest.fn(),
```

Comment:
It would be great to return the value it gets too, so I can remove my custom mock in a11y 🙂. Otherwise, we won't have any text being rendered.

https://github.com/liferay/liferay-portal/blob/5f33f077a3f08ca5c2e9872d6f9d42c234e47a37/modules/apps/frontend-js/frontend-js-a11y-web/jest-setup.config.js

Reply:
```suggestion
	sub: jest.fn((str) => str),
```

Looks okay?

Reply:
Yes, it would be good to just test to see if this won't break some Portal test that wasn't expecting the return of the `sub`, in case it breaks if it's just a snapshot I think we could just update.